techniques/systemSettings/systemManagement/fstabConfiguration/4.0/metadata.xml

```diff
@@ -14,14 +14,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<!--
-    This is the fstab configuration PT.
-    Compatibility : UNIX like
-
-It is intended to check if the fstab is up to date with the
-given parameters.
--->
-<TECHNIQUE name="Fstab configuration">
+<TECHNIQUE name="Mountpoints (fstab)">
```

Comment:
That's two words: "Mount points". But shouldn't this be "Filesystem mount points"? Since we act on filesystems, and mount points are the settings?